lib/plugsnag.ex

```diff
@@ -32,7 +32,8 @@ defmodule Plugsnag do
       end
 
       defp redact(params) do
-        filtered = redact_config.fields
+        fields = redact_config[:fields] || ~w(password)
```

Comment:
I'm not sure about that. These are simple defaults when there's no configuration at all. Our rules should always override them.